### PR TITLE
Push to appcatalog if its tagged

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/giantswarm/issues/7029

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,131 +1,17 @@
...
-    steps:
-    - checkout
-
-    - attach_workspace:
-        at: .
-
-    - run: ./architect publish --stable
+version: 2.1
+orbs:
+  architect: giantswarm/architect@0.2.0
```
[the remark this answers is not in the document]
Comment (tomahawk28) [reply .circleci/config.yml]: Surely I will follow as next PR.

File: helm/node-exporter-app/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 appVersion: "v0.18.0"
 description: A Helm chart for Kubernetes Node-Exporter Service
-name: kubernetes-node-exporter-chart
+name: node-exporter-app
```
[the remark this answers is not in the document]
Comment (tomahawk28) [reply helm/node-exporter-app/Chart.yaml]: That's my bad, I will change that as well.

File: .circleci/config.yml (hunk 1 above)
Comment (kopiczko) [inline .circleci/config.yml]: architect orb is at version `0.4.0` ATM.

Comment (rossf7) [reply .circleci/config.yml]: @kopiczko Thanks I'll fix. Some other repos might need updating. 

Added a note to my board.

Comment (kopiczko) [reply .circleci/config.yml]: Thanks. BTW it's 0.4.1 since yesterday :)

**Pull request merged**